### ISSUE TITLE
Dual Merchant

### ISSUE THREAD

**Issue by MCAProTools** [MCAProTools/main#1]
The client has two Authorize.net accounts. The account #1 was used from May 15th forward. Which has 900+ subscriptions. 

The client, got approved for Authorize.net account #2 which has a lower APR / FEE - so any NEW subscriptions moving forward should be processed through Authorize.net #2.  

 …HOWEVER, the original subscriptions from May 15th still need to remain on Authorize.net #1 account for “renewals”. 

So basically there is going to be two different payment methods. One for the old subscriptions and the other one would be for newer subscriptions moving forward.

Any "New" One Time Payments would also always go through Authorize 2 account. 

**Ask for Login information to Merchant.**
